pkg/kubelet/kubelet.go

```diff
@@ -1052,7 +1006,7 @@ func (kl *Kubelet) retrieveNodeLabelsFile(path string) (map[string]string, error
 
 	err = yaml.NewYAMLOrJSONDecoder(bufio.NewReader(fd), 12).Decode(&kps)
```

Comment:
Where is the format for this file defined? Is it versioned? How can we introduce changes to this format?